WordPress/openverse
# Update links in metadata section on single result page

## Fixes



Fixes #2731 by @obulat

## Description


This PR updates the metadata section of the single result page.

The metadata items match the updates styles: [Metadata component in Figma](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?type=design&node-id=5171-16563&mode=design&t=blxAwawVa4wBfeT4-4), [Metadata in the audio page in Figma](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?type=design&node-id=5645-29968&mode=design&t=wlOZdKnd7mwKmcQa-4)

The items don't wrap anymore, unless they don't fit. You can see that in the snapshot comparisons: previously, the long values would be split between 2-3 lines, and now they are all on one line.

The external links to the source, the provider and the album have an "external link" icon added to them.

The "source" link is updated to link to the source's landing page instead of the media's foreign landing URL.


## Testing Instructions

Check that the single result page metadata section matches the mockups. The code should make sense, too.
When you click on the links, correct analytics events should be fired, and only one per link (not external link click + some other event).

_Note:_ the (i) information popover explaining the difference between provider and source is not in scope of this PR

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
